vignettes/GOstatsHyperG.Rmd

```diff
@@ -0,0 +1,521 @@
+
+pvalFiltered <- yourData[hasSmallPvalue, ]
+selectedEntrezIds <-unlist(mget(featureNames(pvalFiltered),
+                                hgu95av2ENTREZID))
+```
+
+Here is a description of all the arguments needed to construct a
+*GOHyperGParams* instance.
+
+:::{.description}
+| **geneIds** A vector of gene identifiers that defines the selected list of genes. This is often the output
+|           of a test for differential expression among two sample groups. For experiments using Affymetrix
+|           expression arrays, this should be a vector of Entrez Gene IDs. If you are using the
+|           **YEAST** annotation package, the vector will consist of yeast systematic names.
+
+| **universeGeneIds** A vector of gene identifiers that defines the universe of possible 
+|           genes. We recommend using the set of gene IDs that result from non-specific filtering. 
+|           The identifiers should be of the same type as the `geneIds`; for Affymetrix arrays, these will
+|           be Entrez Gene IDs.
+
+| **annotation** A string giving the name of the annotation data package that corresponds to the chip
+|           used in the experiment.
+
+| **ontology** A two-letter string specifying one of the three GO ontologies: BP, CC, or MF. 
+|          The `hyperGTest` function only tests a single GO ontology at one time.
+
+| **pvalueCutoff** A numeric values between zero and one used as a $p$-value cutoff for $p$-values
+|         generated by the Hypergeometric test. When the test being performed is non-conditional, this is
+|         only used as a default value for printing and summarizing the results. For a conditional
+|         analysis, the cutoff is used during the computation to determine perform the conditioning: child
+|         terms with a $p$-value less than `pvalueCutoff` are conditioned out of the test for their parent term.
+
+| **conditional** A logical value. If `TRUE`, the test performed uses the conditional algorithm. Otherwise,
+|         a standard Hypergeometric test is performed.When 'conditional(p) == TRUE', the 'hyperGTest'
+|         function uses the structure of the GO graph to estimate for each term whether or not there is
+|         evidence beyond that which is provided by the term's children to call the term in question
+|         statistically overrepresented. The algorithm conditions on all child terms that are themselves
+|         significant at the specified p-value cutoff. Given a subgraph of one of the three GO ontologies,
+|         the terms with no child categories are tested first. Next the nodes whose children have already
+|         been tested are tested. If any of a given node's children tested significant, the appropriate
+|         conditioning is performed.
+
+| **testDirection** A string which can be either "over" or "under". This determines whether the test
+|         performed detects over or under represented GO terms.
+:::
```

Comment:
This section wouldn't be so easy to maintain because of its formatting. I think it would be better to remove `:::{.description}` and the ending `:::` then do for each attribute:

    `geneIds` A vector of gene identifiers that defines the selected list of genes. This is often the output
    of a test for differential expression among two sample groups. For experiments using Affymetrix
    expression arrays, this should be a vector of Entrez Gene IDs. If you are using the
    `r Biocpkg("yeast")` annotation package, the vector will consist of yeast systematic names.

Note: I saw the yeast package mentioned; however, I am not sure which one they are referring to.